sklearn/multioutput.py

```diff
@@ -233,6 +233,18 @@ class MultiOutputRegressor(RegressorMixin, _MultiOutputEstimator):
     ----------
     estimators_ : list of ``n_output`` estimators
         Estimators used for predictions.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.datasets import load_linnerud
+    >>> from sklearn.multioutput import MultiOutputRegressor
+    >>> from sklearn.linear_model import Ridge
+    >>> np.random.seed(123)
```

Comment:
Usually we don't fix the random state in this way. This better to pass the random state in the constructor in the example:

```python
Ridge(random_state=123)
```